packages/markdown/processor/plugin/table-flattening.js

```diff
@@ -9,10 +9,11 @@ function transformer(ast) {
       // This is necessary to pullout all the relevant strings
 
       // Parse Header Values
-      const headerChildren = header.children[0].children;
+      const headerChildren = header.children && header.children.length ? header.children[0].children : [];
```

Comment:
Are we allowed to use the null coalescing operator yet? Syntax is still new to me, but would this work?

```suggestion
      const headerChildren = header?.children?.[0].children || [];
```

Reply:
@erunion do you know?

Reply:
Optional chaining was turned on by default in `@babel/preset-env` 7.8, and we're on 7.9 so you should be good to use it! Would recommend looking at the compiled dist and making sure it's translated okay though.

https://github.com/babel/babel/pull/10811

Reply:
I'll give it a shot!

Reply:
Assuming it's good, once ESLint or [eslint-plugin-unicorn](https://www.npmjs.com/package/eslint-plugin-unicorn) get a rule for it, we can probably start to enforce and prefer optional chaining

Reply:
Yeah we definitely don't support it in our linting at this moment. Might want to include it [here](https://github.com/readmeio/eslint-config/blob/master/index.js) going forward?

Reply:
Ah nevermind, I see what you're saying. 
> Once it's included. 
Let's hold off on moving this until we can lint enforce it.

Reply:
good by me 👍

Reply:
Yeah I meant you should be good to use it now, just that once it gets added into ESLint we'll start enforcing it in our apps that use Babel.

Reply:
I got ya. Let's just do it all in one sweep!